internal/commands/ebpf/profiling/createByFixedTime.go

```diff
@@ -0,0 +1,103 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package profiling
+
+import (
+	"time"
+
+	"github.com/apache/skywalking-cli/internal/flags"
+	"github.com/apache/skywalking-cli/internal/model/ebpf"
+	"github.com/apache/skywalking-cli/pkg/display"
+	"github.com/apache/skywalking-cli/pkg/display/displayable"
+	ebpf_graphql "github.com/apache/skywalking-cli/pkg/graphql/ebpf"
+
+	"github.com/urfave/cli/v2"
+
+	api "skywalking.apache.org/repo/goapi/query"
+)
+
+var CreateCommand = &cli.Command{
+	Name:    "createByFixedTime",
```

Comment:
I've never saw such kind of command names. If you think there might be other kinds of creation, consider using a subcommand like `create fixed`

Reply:
Update to `create fixed`